docs/source/tutorials_source/platform/tutorial_active_learning_detectron2.py

```diff
@@ -0,0 +1,306 @@
+"""
+
+.. _lightly-tutorial-active-learning-detectron2:
+
+Tutorial 4: Active Learning using Detectron2 on Comma10k
+=========================================================
+
+In machine learning projects we often don't start training a model from scratch
+but we start with an already pre-trained model. For object detection tasks a
+common dataset is MS COCO consisting of over 100'000 images containing 80
+different classes. Our goal is now to take a MS COCO pretrained model and optimize
+it for an autonomous driving task where we only care about pedestrians and cars
+(two classes present within MS COCO). We use the pretrained model and run
+predictions on a second dataset (Comma10k) which has been collected for
+autonomous driving. The predictions combined with self-supervised embeddings we 
+can get using the lighty framework allow us to find the 100 most important images
+through active learning. Active learning is a process where we take model predictions
+to find new images for annotation having a great positive impact on model accuracy.
+
+
+This tutorial is available as a
+`Google Colab Notebook <https://colab.research.google.com/drive/1r0KDqIwr6PV3hFhREKgSjRaEbQa5N_5I?usp=sharing>`_
+
+In this tutorial you will learn:
+
+*   how to use Lightly Active Learning together with the
+    `detectron2 <https://github.com/facebookresearch/detectron2>`_ framework
+    for object detection
+*   how to use the Lightly Platform to inspect the selected samples
+*   how to download the selected samples for labeling
+
+The tutorial will be divided into
+the following steps. 
+
+#. Installation of detectron2 and lightly
+#. Run predictions using a pretrained model
+#. Use lightly to compute active learning scores for the predictions
+#. Use the Lightly Platform to understand where our model struggles
+#. Select the most valuable 100 images for annotation
+
+Requirements
+------------
+- Make sure you have the detectron2 framework installed on your machine. Check out
+  the `detectron2 installation documentation <https://detectron2.readthedocs.io/en/latest/tutorials/install.html>`_
+
+- In this tutorial we work with the comma10k dataset. The dataset consists of
+  10'000 images for autonomous driving and is available
+  `here on GitHub <https://github.com/commaai/comma10k>`_
+  We can download the dataset using `git clone`. We save the dataset locally
+  to `/datasets/`
+
+.. code::
+
+    git clone https://github.com/commaai/comma10k
+
+"""
+
+# Some basic setup:
+# Setup detectron2 logger
+import detectron2
+from detectron2.utils.logger import setup_logger
+setup_logger()
+
+# import some common libraries
+import numpy as np
+import os, json, cv2, random, glob
```

Comment:
Here you require cv2 and it's not mentioned that you have to install it anywhere. Additionally, you say above that the first step will be to show how lightly is install yet there is no information on that.

Reply:
Good catch. I'll add the info about installing opencv